app/javascript/utilities/connect/newMessageNotify.js

```diff
@@ -0,0 +1,19 @@
+export default function notifyUser() {
+  modifyTitle();
+  // May need it's own big feature
+  // const audio = new Audio('../../../assets/sound/notification.mp3');
+  // audio.play();
+}
+
+const modifyTitle = () => {
```

Comment:
I'm not sure I follow the logic here.

- Every 2 seconds, check if the title of the document is the initial value of the document title and if so change it to "New Message 👋🏻", otherwise set it back to the original title.
- And let that all run for 12 seconds and then stop it.

Any reason we do 12 seconds and also, why do we stop? Don't we want to keep notifying users that they have new messages if new ones come in?

Reply:
> Any reason we do 12 seconds and also, why do we stop? Don't we want to keep notifying users that they have new messages if new ones come in?

So 12 sec only because I thought it will be too disturbing to keep doing it. It's like it's shouting for your attention even if I have seen it. If there is a new message the notification will start again for 12 sec. But if you think that we should keep it going that's also fine.

Reply:
Ahh, that's the part I misunderstood. I didn't realize it was 12 seconds for only when a new message comes in. In that case, instead of hardcoding `12000` milliseconds, consider making it a constant explaining what it is.